contracts/deploy/Create.sol

```diff
@@ -7,12 +7,12 @@ pragma solidity ^0.8.0;
  * @notice This contract deploys new contracts using the `CREATE` opcode and is used as part of
  * the `Create3` deployment method.
  */
-contract CreateDeployer {
+contract Create {
     /**
      * @dev Deploys a new contract with the specified bytecode using the CREATE opcode.
      * @param bytecode The bytecode of the contract to be deployed
      */
-    function deploy(bytes memory bytecode) external {
+    function createDeploy(bytes memory bytecode) external {
```

Comment:
```suggestion
    function deploy(bytes memory bytecode) external {
```
seems more intuitive?

Reply:
we have `deploy` in almost every contract here and it means different things. So maybe makes sense to separate concerns?

Reply:
i think the contract name should contextualize the method. Otherwise, we'd end up prefixing error/methods for many of our util contracts. This contract isn't being inherited from anyways, so should be fine?